node/cli/src/cli.rs

```diff
@@ -100,15 +100,6 @@ pub struct Cli {
 	#[cfg(feature = "frequency-no-relay")]
 	#[clap(long, help = "The interval in seconds", default_value = "120", value_name = "SECONDS")]
```

Comment:
I think we want the default interval to be 12 seconds not 120 seconds

Reply:
done